Commit proposals after running script in R

Mothluin opened the pull request:
I stumbled upon your blog post and decided to give it a shot.
I'm still learning R but I think I found some problems in your code and managed to fix these by trial and error.
1. It appears to me you need to (install and) enable additional libraries on top of the ones you listed. Namely, you need: magrittr, bindrcpp, dplyr. This seems to be essential to make R understand the "%>%"
2. Function "parse_datetime" does not look like a valid function name. My researches hint you probably wanted to pick "parse_date_time". After correcting the typo, R threw another error at me for missing arguments. I've therefore replaced your function with another one. Seems to work fine
3. The code used to stitch together the 3 graphs delivers a plot with no bars for me. I played around with the syntax and managed to display the graph correctly after removing the "ncol = 3, scale = 0.7" part

In your blog post you also imported package "ggpthemes", which does not exist: this was probably a typo for "ggthemes".
In any case, I saw you already amended that here on GitHub.

Mothluin committed:
Commit proposals after running script in R

I stumbled upon your blog post and decided to give it a shot.
I'm still learning R but I think I found some problems in your code and managed to fix these by trial and error.
1. It appears to me you need to (install and) enable additional libraries on top of the ones you listed. Namely, you need: magrittr, bindrcpp, dplyr. This seems to be essential to make R understand the "%>%"
2. Function "parse_datetime" does not look like a valid function name. My researches hint you probably wanted to pick "parse_date_time". After correcting the typo, R threw another error at me for missing arguments. I've therefore replaced your function with another one. Seems to work fine
3. The code used to stitch together the 3 graphs delivers a plot with no bars for me. I played around with the syntax and managed to display the graph correctly after removing the "ncol = 3, scale = 0.7" part

In your blog post you also imported package "ggpthemes", which does not exist: this was probably a typo for "ggthemes".
In any case, I saw you already amended that here on GitHub.

amrrs commented:
Thanks very much for the PR. Currently I'm traveling I'll check it Tomo and act !

Mothluin commented:
I went back to the script and tested again the plot_grid call to stitch the 3 plots together.
I now managed to get the graphs displayed while keeping the "ncol" option (which makes sense to keep, after all).

Mothluin committed:
filter() not needed

The "filter(as.Date(time_ist) >= as.Date("2017-01-01"))" call is not required because you've already discarded all data prior 2017-01-01 at line 21.
